src/password_generator.rs

```diff
@@ -0,0 +1,27 @@
+use rand::Rng;
+
+pub fn password_generator(length: usize, category: usize) -> String {
+    let mut rng = rand::thread_rng();
+    if category == 0 { 
+        let password_chars: String = (0..length)
+            .map(|_| {
+                let ascii_val = rng.gen_range(33..=126);
+                ascii_val as u8 as char
+            })
+            .collect();
+    password_chars
```

Comment:
This is a total nit, but no need to declare a variable and then just return it, just return the value directly when you generate it.